### PR TITLE
Revert "Hardcode predictor config to workaround parsing issue"

### PR DESCRIPTION
**Pull request by fridex**
Reverts thoth-station/thoth-application#448

### CODE REVIEW

**Comment by fridex** [comment]
/hold

let's merge this once user-api will have fix for thoth-common, see comment in https://github.com/thoth-station/adviser/issues/1230#issuecomment-694003970

**Review by harshad16 — APPROVED** [review]

**Pull request merged**
